ansible/roles/beetbox-drupal/templates/uploadprogress.ini.j2

```diff
@@ -0,0 +1,2 @@
+; activate and configurate for php uploadprogress module
```

Comment:
Another option would be to use -- http://docs.ansible.com/ansible/blockinfile_module.html